app/components/classCard.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { View, Text, Image, StyleSheet, TouchableOpacity } from 'react-native';
+import Colors from '../utils/Colors';
+
+interface ClassCardProps {
+  time: string;
+  title: string;
+  members: string;
+  year: string;
+  status: string;
+  imageUrl: string;
+  onPress: () => void;
+}
+
+const ClassCard = ({ time, title, members, year, status, imageUrl, onPress }: ClassCardProps) => {
+  return (
+    <TouchableOpacity style={styles.card} onPress={onPress}>
+      <Image source={{ uri: imageUrl }} style={styles.image} />
+      <View style={styles.info}>
+        <Text style={styles.time}>{time}</Text>
+        <Text style={styles.title}>{title}</Text>
+        <View style={{ flexDirection: 'row', alignItems: 'center' }}>
+          <View style={styles.dot} />
+          <Text style={styles.members}>{members}</Text>
+        </View>
+        <Text style={styles.year}>
+          <Text style={{ fontWeight: 'bold' }}>Jahrgangsstufe:</Text> {year}
+        </Text>
+        <Text style={styles.status}>
+          <Text style={{ fontWeight: 'bold' }}>Status: </Text>{status}
+        </Text>
+      </View>
+    </TouchableOpacity>
+  );
+};
+
+const styles = StyleSheet.create({
+  card: {
+    flexDirection: 'row',
+    backgroundColor: Colors.surface,
+    borderRadius: 8,
+    marginVertical: 15,
+    marginHorizontal: 30,
+    overflow: 'hidden',
+  },
+  image: {
+    width: 140,
+    height: 140,
+  },
+  info: {
+    flex: 1,
+    paddingLeft: 30,
+    paddingVertical: 20,
+  },
```

Comment:
Instead of hardcoding the image dimension make use of the ``flex`` attribute. Remove the ``width`` and ``height`` from image and instad put ``flex: 1``. And the on the ``info`` class you change the ``flex: 1`` to ``flex: 1.5``. The flex number determines the ratio of how those two elements share space in their parent element.

Reply:
done